app/helpers/change_description_helper.rb

```diff
@@ -0,0 +1,88 @@
+module ChangeDescriptionHelper
```

Comment:
sure @differentreality I will change the name of the helper and arrange functions accordingly.